DataReader/Controller/ViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ViewController.swift
+//  DataReader
+//
+//  Created by Damian Stanczyk on 23.10.2017.
+//  Copyright © 2017 haze. All rights reserved.
+//
+
+let numOfCells = 10
+
+import UIKit
+
+class ViewController: UIViewController, UITableViewDataSource {
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    // TODO: textObjects should be private
+    var textObjects = [String]()
+    private let reuseCellId = "DynamicCell", cellNibName = "DynamicCell"
```

Comment:
duplicate variable? with the same value?

Reply:
Of course its the same value in this case, however one is for xib name and next for cell identifier which may differ and thats the reason why I have two variables

Reply:
but the aren't ;)

Reply:
haha right